poster/styles/styles.scss

```diff
@@ -1,63 +1,477 @@
-.u-text_main {
+// utility //
+
+.u-text_details {
     font-size: .7rem;
     margin-left: 1rem;
     margin-right: 1rem;
 }
 
+.u-remove-list {
+    list-style: none;
+    list-style-type: none;
```

Comment:
`list-style` is a shorthand property that includes `list-style-type`, so you don't need both lines 10 and 11. In general, it's my recommendation to only write (or imply) the code you want, so I'd keep line 11 and delete line 10. For anything you don't declare inside the shorthand property, it resets things back to their default/initial value (unless you override it later). You can learn more about the [`list-style` shorthand here](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style) and about [shorthands in general](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties).